### PR TITLE
Rs 2022.09.18

### PR DESCRIPTION
**Pull request by rschlaikjer**
2022-09-18-162057_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 7/07_01_CIEL07_4.txt [hunk 1]
```diff
@@ -6,53 +6,59 @@ Dinner goes by without incident.
...
+I'll sneak out of here once everyone's gone to bed,
+ so for the time being I should just act casual and try not to rouse any suspicions.
 }
 [sha:b4aaac27715e5e2db164a56f34b29b80d76ef149]{
 -- Page 399, Offset 15771.
 -- 「んー、ダージリンでしたら二番目より一番目の方が好まれると思いますよ？　味も香りも控えめで後に残りませんもの」
 // 二番目 and 一番目 here are the same concept as the first/second flush mentioned below, but I've left them distinct as in the original.
-"Hmm, I believe he'd appreciate Darjeeling more if it was the first picking rather than the second?
- Both the taste and the aroma is more subdued, without the same aftertaste."
+"Hmm, I believe he'd appreciate Darjeeling more if it were the first picking rather than the second?
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_4.txt:34]: This question mark is probably not needed in English.

File: script/Ciel/Day 7/07_01_CIEL07_4.txt [hunk 2]
```diff
@@ -103,38 +111,43 @@ Even though I'm not even part of the conversation, Kohaku suddenly gives me the
...
+ Am I mistaken?"
 }
 [sha:dc285b32d02bbe664331dc5361f39ea97ba7e123]{
 -- Page 408, Offset 15789.
 -- 「もちろん正しい。けど、いくらなんでも８時っていうのは早すぎないか？　子供じゃないんだから、この際10時ぐらいに決めなおそう」
 "You're right, of course.
- That said, no matter how you look at it, wouldn't you say eight o'clock is far too early?
- I'm not a kid, so why don't we take this opportunity and rearrange to be at least ten o'clock?"
+ That said, surely even you can agree that eight o'clock is way too early?
+ I'm not a kid, so why don't we take this opportunity to push that time back to ten o'clock?"
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_4.txt:143]: Still feel like 'at least' is appropriate

File: script/Ciel/Day 7/07_01_CIEL07_4.txt [hunk 3]
```diff
@@ -149,20 +162,21 @@ When she puts it like that, there's not much room for a rebuttal.
...
+If someone with as few commitments as myself isn't returning home early,
+ there isn't much they could be off doing other than fooling around.
 }
 [sha:50bef6066fc91e27d01e58fb70da6bcaf6ee1d7a]{
 -- Page 411, Offset 15794.
 -- 「もっとも、門限を守ればいい、という話ではありませんけど。これまでの自堕落ぶりを反省して早く帰ってきてくれたのは喜ばしいですが、その後が気になります。夕食から心ここにあらずという様子ですが、何かあったのですか？
-"Although, that doesn't mean all you have to do is follow the curfew.
- I am pleased that you've reflected on your self-indulgence up until now and returned home early, but I'm still concerned about what comes thereafter.
- You appear to have had your mind elsewhere ever since dinner――did something happen?"
+"Of course, adhering to the curfew isn't the only thing I expect of you.
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_4.txt:171]: I feel like 'Although' conveys the meaning better here.

**Review by Petrikow — COMMENTED** [review]
otu

File: script/Ciel/Day 7/07_01_CIEL07_4.txt [hunk 4]
```diff
@@ -103,38 +111,43 @@ Even though I'm not even part of the conversation, Kohaku suddenly gives me the
...
+ I mean, whenever I mess something up she immediately takes care of it for me,
+ and when I'm feeling under the weather but haven't picked up on it yet she points it out for me.
 }
 [sha:ad4cc9b9d148aa75bde23d90627b5fdd0e9935b9]{
 -- Page 406, Offset 15787.
 -- あと夜遅くに帰っても誰かさんみたいに怒らないし。そういうのはおとなしい、優しい、と表現すべきだよ」
-And finally, whenever I come home late at night she doesn't get mad at me, unlike a certain somebody.
- That's more appropriately expressed as 'well-behaved' and 'kind', honestly."
+On top of that, she doesn't get mad at me when I come home late in the evening, unlike a certain somebody.
+ I think you'd call that 'well-behaved' and 'kind', if anything."
```
Comment (yk1512) [inline script/Ciel/Day 7/07_01_CIEL07_4.txt:128]: I would replace 'well behaved' with gentle. Feels a little arrogant for Shiki to say.

File: script/Ciel/Day 7/07_01_CIEL07_4.txt [hunk 5]
```diff
@@ -173,44 +187,46 @@ Do you, for example... have some sort of plans after this?
...
+I'm this close to breaking eye contact with her, but stop myself at the last moment.
 }
 [sha:77277f27ece7ea91f635cd07fe7890689e5b98b4]{
 -- Page 413, Offset 15798.
 -- 秋葉のヤツ、なんて勘の良さか。そしてその<疑|ネ><惑|タ>をここぞというタイミングで切り出すとか、いつもの俺なら一刀両断に切られてボロをだしていたに違いない。
-// 一刀両断に切られて here has been dramatically shortened to "immediately", maybe there's a better way of expressing it, but being literal is obviously incorrect.
-How is Akiha %{i}this%{/i} perceptive?
- To think she'd bring out these kind of suspicions at this critical juncture――My usual self would've no doubt given myself away immediately.
+How is Akiha %{i}so%{/i} damn perceptive?!
+ She hit me with these allegations right at this critical juncture, too――ordinarily I'd have folded like a house of cards.
```
Comment (yk1512) [inline script/Ciel/Day 7/07_01_CIEL07_4.txt:196]: don't feel like a comma is needed here

File: script/Ciel/Day 7/07_01_CIEL07_4.txt [hunk 6]
```diff
@@ -230,25 +246,29 @@ At my outright flagrantly calm composure, Akiha awkwardly breaks eye-contact.
...
+ It's possible it has something to do with the river running through the garden...?
+ What do you think, Kohaku?"
 }
 [sha:4c814a63d110a568c72d309d5139de01a2d36fe1]{
 -- Page 419, Offset 15809.
 -- 「わたしも今のところ気になる故障はありませんけど……ああ、もしかして地下の古い水路のことかもしれません。
-"There have been nothing out of order I've noticed as of late, at least...
- Oh, perhaps it's the old waterway down underground?
+"There aren't any problems with it that I'm aware of at the moment either, but...
+ Ah, there's a chance it could be related to the old aqueduct underground?
```
Comment (yk1512) [inline script/Ciel/Day 7/07_01_CIEL07_4.txt:265]: Ah, perhaps it could be...

File: script/Ciel/Day 7/07_01_CIEL07_4.txt [hunk 7]
```diff
@@ -259,28 +279,30 @@ That's all, apparently.
...
 -- 水路の話は終わり、何でもない会話に戻る。
-The subject of the waterway comes to an end, and we return to talking about nothing in particular again.
+The subject of the aqueduct comes to an end, and we return to talking about nothing in particular again.
 }
 [sha:b88104916e3ad737ac6d8cce8749a2e5210796a6]{
 -- Page 422, Offset 15815.
 -- 結局お茶会は８時過ぎまで続いたが、俺が入浴する事でお開きになった。
-Ultimately, the tea party goes on until eight o'clock, with me taking my leave on account of the bath.
+The tea party ends up carrying on past eight o'clock,
+ though I have to excuse myself to go take a bath.
```
Comment (yk1512) [inline script/Ciel/Day 7/07_01_CIEL07_4.txt:300]: > The tea party ends up carrying on past eight o'clock, though I **had** to excuse myself to go take a bath.

Sounds to me like the 'taking bath' is a past event that he's commenting on, while the 'tea party' is supposedly still ongoing.

**Review by yk1512 — APPROVED** [review]
🙏

File: script/Ciel/Day 7/07_01_CIEL07_4.txt [hunk 8]
```diff
@@ -6,53 +6,59 @@ Dinner goes by without incident.
...
-My emotions are in high-gear because of it, but fretting over it right now won't accomplish anything.
+I've got butterflies in my stomach just thinking about it,
+ but sitting here fretting isn't going to accomplish anything.
 }
 [sha:145a17d8c279611dc8d828792642b508aeb905fa]{
 -- Page 398, Offset 15770.
 -- 就寝時間になってからこっそり出かける事になるので、今は怪しまれないよう余裕を持って過ごさなければ。
-I'll sneak out once its bedtime, so right now I should just spend my time leisurely as to not rouse any suspicions.
+I'll sneak out of here once everyone's gone to bed,
+ so for the time being I should just act casual and try not to rouse any suspicions.
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_4.txt:28]: being, I should just

File: script/Ciel/Day 7/07_01_CIEL07_4.txt [hunk 9]
```diff
@@ -6,53 +6,59 @@ Dinner goes by without incident.
...
 -- 「いえいえ、翡翠ちゃんはお茶にはこだわりませんよ。
-"Not at all, Hisui-chan does not have any particular preferences when it comes to tea.
+"Not necessarily; Hisui-chan doesn't have any particular preferences when it comes to tea.
 }
 [sha:829f29c761f932d41f564a5878a295ea0dd058b5]{
 -- Page 401, Offset 15775.
 -- 神経質そうに見えますけど、こう見えてわりとず―――」
 // not sure what the ず that gets interrupted here is supposed to be, so I've sort of left it untranslated.
-She might appear fussy, but contrary to expectations she's a rather-"
+She might appear fussy, but contrary to expectations she's a rather―――"
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_4.txt:61]: very fussy looking
image

File: script/Ciel/Day 7/07_01_CIEL07_4.txt (hunk 3 above)
Comment (Avuxo) [reply script/Ciel/Day 7/07_01_CIEL07_4.txt:171]: "Of course" holds the meaning imo and sounds way more akiha-like

File: script/Ciel/Day 7/07_01_CIEL07_4.txt (hunk 7 above)
Comment (Avuxo) [reply script/Ciel/Day 7/07_01_CIEL07_4.txt:300]: if that's the case (and i can certainly see why it would be), it would be "ended up" on top of "had".

**Review by Avuxo — APPROVED** [review]

**Pull request merged**